src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
```

Comment:
Shouldn't this go to https://github.com/dotnet/runtime/blob/main/src/coreclr/System.Private.CoreLib/src/ILLink/ILLinkTrim.xml ?
The libraries folder is shared between the runtimes, so anything runtime specific should ideally go to the runtime specific folder.